### PR TITLE
Default to in-memory storage for the proxy

### PR DESCRIPTION
**Pull request by carolynvs**
This makes it easier for someone to try out or just run the proxy by executing the binary without further environment setup.

### CODE REVIEW

File: cmd/proxy/actions/storage.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 // GetStorage returns storage backend based on env configuration
 func GetStorage() (storage.BackendConnector, error) {
 	// changing to mongo storage, memory seems buggy
```
Comment (rchakra3) [inline cmd/proxy/actions/storage.go]: nit- maybe get rid of this comment?

Comment (rchakra3) [reply cmd/proxy/actions/storage.go]: As per https://github.com/gomods/athens/issues/165 it looks like the issue was fixed!

Comment (robjloranger) [reply cmd/proxy/actions/storage.go]: Awesome, so just comment clean up :+1:

**Comment by michalpristas** [comment]
I'm so looking forward to this @carolynvs 
On this, I believe we should put new community members first. I believe we are able of setting one env var to tweak behavior.

**Comment by marwan-at-work** [comment]
@michalpristas 100%. I was just thinking it might be a bad experience if they're not aware that we're storing modules in their memory by default. So @carolynvs 's docs are gonna be perfect 💯

**Comment by robjloranger** [comment]
this is going to be great @carolynvs, that third install option will go nicely with the little server under my desk :grinning:

**Comment by carolynvs** [comment]
hehe I _may_ have some experience with deploying things under my desk to avoid IT 😇

**Comment by carolynvs** [comment]
Yup! I will call out the OOM and other concerns of running with in-memory storage in follow-up doc PRs.

**Comment by marwan-at-work** [comment]
@carolynvs @arschles I love all the lengthy explanations from everyone. Sometimes it's hard to be aware of all the scenarios so they really help :)

**Pull request merged**